incident/api/serializers.py

```diff
@@ -327,4 +321,3 @@ class CSVIncidentSerializer(VariableFieldSerializer):
     links = serializers.CharField(source='link_summary')
     equipment_broken = serializers.CharField(source='equipment_broken_summary')
     equipment_seized = serializers.CharField(source='equipment_seized_summary')
-    status_of_charges = serializers.CharField(source='status_of_charges_summary')
```

Comment:
I think we can keep this here, as `status_of_charges_summary` does not use the removed `status_of_charges` field and instead uses the new style charge metadata.